libs/core/kiln_ai/adapters/ml_model_list.py

```diff
@@ -1225,6 +1225,12 @@ class KilnModel(BaseModel):
                 supports_structured_output=False,
                 supports_data_gen=False,
             ),
+            KilnModelProvider(
+                name=ModelProviderName.docker_model_runner,
+                model_id="ai/llama3.2",
```

Comment:
We should set the size explicitly, so if the default ever changes, our UI is still correct. `llama3.2:3B-Q4_K_M`?

We should also add the rest of the models. I know the list is bit and ugly... but it's actually really good for usability. Details: https://getkiln.ai/blog/i_wrote_2000_llm_test_cases_so_you_dont_have_to

Adding the models docker supports in there will do a few nice things:
 - Nice names in the UI
 - Add them to the test suite
 - Once tested that they will appear higher in the dropdown as we'll trust them to perform certain tasks better (JSON mode, complex requests like synthetic data, etc).

Reply:
I have update the model references to include a more explicit tag `<weight>-<quantization>`
I considered adding all supported models as well, but I thought it might be better to include them in a follow-up PR that focuses solely on listing the supported models.

That said, if you’d prefer to include them now, I’m happy to do so.